SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -102,6 +100,19 @@ const Worker_ComponentId STARTUP_ACTOR_MANAGER_COMPONENT_ID = 9993;
 const Worker_ComponentId GSM_SHUTDOWN_COMPONENT_ID = 9992;
 const Worker_ComponentId HEARTBEAT_COMPONENT_ID = 9991;
 
+const Worker_ComponentId SERVER_AUTH_COMPONENT_SET_ID = 9900;
```

Comment:
Nit (applies for all of these)
```suggestion
const Worker_ComponentSetId SERVER_AUTH_COMPONENT_SET_ID = 9900;
```